src/sabre/addressing.ts

```diff
@@ -0,0 +1,58 @@
+// Copyright 2020 Cargill Incorporated
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import crypto from 'crypto';
+
+// The namespace registry prefix for global state (00ec00)
+const NAMESPACE_REGISTRY_PREFIX = '00ec00';
+
+// The contract prefix for global state (00ec02)
+const CONTRACT_PREFIX = '00ec02';
+
+// The contract registry prefix for global state (00ec01)
+const CONTRACT_REGISTRY_PREFIX = '00ec01';
+
+// The Sawtooth Sabre transaction family name (sabre)
+export const SABRE_FAMILY_NAME = 'sabre';
+// The Sawtooth Sabre transaction family version (0.4)
+export const SABRE_FAMILY_VERSION = '0.4';
```

Comment:
Is this going to be incremented with the latest Sabre release? I wonder what that implies about transact's compatibility with the version of Sabre that you have running.